### PR TITLE
Replace /clubs with /events

### LINKED ISSUE #374: Create /events and remove /clubs from the nav bar
I think the hierarchy should be:
- /events
  - /clubs (perhaps renamed to /workshops because that's really what it is)
  - /hackathons

I think the Ball is big enough still to deserve it's own page.

The new /events page should have some examples of what our events are, what to expect from those events and when to expect them.

### LINKED ISSUE #398: Add Expectations Page
Ok so here's my vision
A page that has basics about what to expect about different categories of event, to make it easier for people to plan, especially aimed at improving accessibility. That might be information about for example, how board games is normally a free-form event where you can join a group and play games, pizza is normally served halfway through & will be organised through queueing, etc.
Basically, should have different categories of event, and information about them. Aiming to be ready to freshers next year :)

### PR DESCRIPTION
**Pull request by LMBishop**
Closes #374, closes #398 

The clubs page still needs updating with our latest workshops, and affiliation with GDSC needs removing, but I can't be bothered to do that here

### CODE REVIEW

File: _collections/_events/board-games-nights.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+image: /assets/images/board-games-night.jpg
+order: 5
+---
+
+At least once a term we run a Board Games Night and/or a LAN Party in the School
+of Computer Science! We always have drinks and pizza, with the School providing 
+alcoholic drinks, and usually take up the Atrium and the Sloman Lounge.
+
+While we have our own board games and some consoles, we encourage everybody to 
+bring their own as well!
```
Comment (MattyTheHacker) [inline _collections/_events/board-games-nights.md:12]: blank space at end of line

File: events.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: Developer Student Clubs
```
Comment (MattyTheHacker) [inline events.html:3]: Title need changing?

**Review by MattyTheHacker — CHANGES_REQUESTED** [review]

File: clubs.html [hunk 3]
```diff
@@ -43,7 +43,7 @@ <h1>Projects</h1>
 
 <div class="fadein-container">
   {% assign session = site.clubs.last %}
-  <div class="club recent fadein">
+  <div class="club recent fadein" >
```
Comment (MattyTheHacker) [inline clubs.html:46]: is there a reason for the extra space or just a typo?

**Review by MattyTheHacker — CHANGES_REQUESTED** [review]

File: clubs.html (hunk 3 above)
Comment (LMBishop) [reply clubs.html:46]: some shoddy merging

**Review by MattyTheHacker — APPROVED** [review]

**Pull request merged**